solcx/install.py

```diff
@@ -359,15 +361,15 @@ def _install_solc_osx(version, allow_osx, show_progress, solcx_binary_path):
     os.chdir(str(temp_path.joinpath("build").resolve()))
     try:
         for cmd in (["cmake", ".."], ["make"]):
-            _check_subprocess_call(cmd, message="Running {}".format(cmd[0]))
+            _check_subprocess_call(cmd, message=f"Running {cmd[0]}")
         temp_path.joinpath("build/solc/solc").rename(binary_path)
     except subprocess.CalledProcessError as e:
         raise OSError(
-            "{} returned non-zero exit status {} while attempting to build solc from the source. "
-            "This is likely due to a missing or incorrect version of an external dependency.\n\n"
-            "You may be able to solve this by installing the specific version using brew: "
-            "https://solidity.readthedocs.io/en/v0.6.0/installing-solidity.html#binary-packages"
-            "".format(cmd[0], e.returncode)
+            f"{cmd[0]} returned non-zero exit status {e.returncode} while attempting"
+            f" to build solc from the source. This is likely due to a missing or"
+            f" incorrect version of an external dependency.\n\n"
+            f"You may be able to solve this by installing the specific version using"
+            f" brew: {CURRENT_RELEASE}"
```

Comment:
The use of `CURRENT_RELEASE` feels a bit misleading to me, since the link points to `v0.6.0` docs which is already six releases old. I'm hesitant to use `latest` though because that page may not always exist in the future.

Maybe something like `INSTALLATION_DOCS` would be a better representation?

Reply:
Sure.  We'll use INSTALLATION_DOCS